client/me/help/help-contact/index.jsx

```diff
@@ -1,642 +0,0 @@
-import config from '@automattic/calypso-config';
-import { getPlanTermLabel } from '@automattic/calypso-products';
-import { Card } from '@automattic/components';
-import { HelpCenter } from '@automattic/data-stores';
-import {
-	SUPPORT_CHAT_OVERFLOW,
-	SUPPORT_FORUM,
-	SUPPORT_HAPPYCHAT,
-	SUPPORT_TICKET,
-	SUPPORT_UPWORK_TICKET,
-} from '@automattic/help-center';
-import { isDefaultLocale, localizeUrl } from '@automattic/i18n-utils';
-import { withDispatch } from '@wordpress/data';
-import debugFactory from 'debug';
-import { localize } from 'i18n-calypso';
-import page from 'page';
-import PropTypes from 'prop-types';
-import { Component, Fragment } from 'react';
-import { connect } from 'react-redux';
-import QueryLanguageNames from 'calypso/components/data/query-language-names';
-import QueryTicketSupportConfiguration from 'calypso/components/data/query-ticket-support-configuration';
-import QueryUserPurchases from 'calypso/components/data/query-user-purchases';
-import HappychatConnection from 'calypso/components/happychat/connection-connected';
-import HeaderCake from 'calypso/components/header-cake';
-import Main from 'calypso/components/main';
-import Notice from 'calypso/components/notice';
-import withActiveSupportTickets from 'calypso/data/help/with-active-support-tickets';
```

Comment:
Looks like this can HoC be deleted.

Reply:
Correct! And it took one more file with it: https://github.com/Automattic/wp-calypso/pull/78433/commits/cf5f36e0e20211fbb4a4ccc8a1034c8a4b3749c8